.github/workflows/tpu-tests.yml

```diff
@@ -73,7 +73,7 @@ jobs:
       shell: bash
 
     # https://docs.github.com/en/actions/deployment/deploying-to-your-cloud-provider/deploying-to-google-kubernetes-engine
-    - uses: google-github-actions/get-gke-credentials@fb08709ba27618c31c09e014e1d8364b02e5042e
+    - uses: google-github-actions/get-gke-credentials@v0
```

Comment:
I set a specific commit because the docs linked right above use a specific commit hash. I'm fine with updating it as long as it works